lib/config.dart

```diff
@@ -56,9 +56,9 @@ String server() {
   if (UniversalPlatform.isWeb) {
     var location = html.window.location;
     if (BeServerLocationOverrides.keys.contains(location.hostname))
-      serverUrl = location.origin + '/';
+			String serverUrl = location.origin ?? '';
```

Comment:
redeclearing `serverUrl` means that BeServerLocationOverrides will no longer work